etstool.py

```diff
@@ -576,5 +587,19 @@ def locate_edm():
     return edm
 
 
+def current_platform() -> str:
```

Comment:
for edm python runtimes you can use the sys.enthought to get the supported platform.
```
>>> sys.enthought
SimpleNamespace(abi='msvc2019', implementation='cpython', build_version='4.1.0.dev18', build_revision='9a43720f6ccc0b7a0cdb5330995f0d989f7fdb12', platform='win_x86_64')
```

Note: I think that the older runtimes used to report the short platform string (e.g. 'win-64')